apps/dashboard/src/routes/(org-site)/courses/+page.svelte

```
@@ -363,75 +355,182 @@
+      <HeroComponent hero={heroProps} orgName={data.org.name} showActions={false} compact={true}>
+        {#snippet navigation()}
+          <NavComponent
+            orgName={data.org.name}
+            logoUrl={data.org.avatarUrl ?? undefined}
+            navItems={landingSettings.navItems}
+            {authAction}
+          />
+        {/snippet}
+        {#snippet children()}
+          {@render searchField()}
+        {/snippet}
```

ESLint

[error] Found an unnecessary children snippet.

(svelte/no-useless-children-snippet)